x-pack/test/functional/apps/aiops/change_point_detection.ts

```diff
@@ -65,15 +65,15 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       await aiops.changePointDetectionPage.selectSplitField(0, 'geoip.city_name');
       await aiops.changePointDetectionPage.getTable(0).waitForTableToLoad();
       const result = await aiops.changePointDetectionPage.getTable(0).parseTable();
-      expect(result.length).to.eql(5);
+      expect(result.length).to.eql(6);
       // assert asc sorting by p_value is applied
       expect(parseFloat(result[0].pValue)).to.be.lessThan(parseFloat(result[4].pValue));
     });
 
     it('allows change point selection for detailed view', async () => {
       await aiops.changePointDetectionPage.getTable(0).selectAllRows();
       await aiops.changePointDetectionPage.viewSelected();
-      await aiops.changePointDetectionPage.assertDetailedView(5);
+      await aiops.changePointDetectionPage.assertDetailedView(6);
```

Comment:
@elastic/ml-ui I'm not totally sure what would have changed in this PR to have warranted this (other than maybe the default pagination size change?) - would appreciate your eyes on this if possible! 🙏